src/ansys/simai/core/data/types.py

```diff
@@ -262,3 +263,10 @@ def get_object_from_identifiable(
         return get_object_from_identifiable(default, directory)
     else:
         raise InvalidArguments(f"Argument {identifiable} is neither a data model nor an ID string.")
+
+
+class SubsetEnum(str, Enum):
```

Comment:
1. Not sure I understand the point of `none` 😕 Does it mean nothing in the body of the request? 
2. It's generally used to have upper case in enumerations:

```py
TRAINING = "Training"
VALIDATION = "Validation"
TEST = "TEST"
```
3. Is it expected that users would select one of those for creating a subset? In this case, should this enumeration be exposed to the users, so that they pull it, something like 
```py
sbts = simai.training_data.subset_names
td.set_subset("wc2y34", sbts.VALIDATION)
```

In this case, class `TrainingDataDirectory` should have a property for it